### PR TITLE
Bug1118201

### PR DESCRIPTION
**Pull request by AutomatedTester**
Update imports to handle current importing process for marionette and the future marionette_driver package that will be released after [Bug 1107336](https://bugzilla.mozilla.org/show_bug.cgi?id=1107336) has landed

### CODE REVIEW

**Comment by try-server-hook** [comment]
[AutomatedTester David Burns (AutomatedTester)](https://github.com/AutomatedTester) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ba09647a9553)

**Comment by try-server-hook** [comment]
[AutomatedTester David Burns (AutomatedTester)](https://github.com/AutomatedTester) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0e46f31682ed)

**Comment by bebef1987** [comment]
Started  http://jenkins1.qa.scl3.mozilla.com/job/flame-kk.ui.adhoc/593/

**Comment by bebef1987** [comment]
We sould also migrate https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_contacts_from_gmail.py#L5

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/test_gallery_crop_photo.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-from marionette.by import By
+try:
+    from marionette.by import By
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/test_gallery_crop_photo.py]: This import in not required in the test and can be removed

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_dsds_cell_data.py [hunk 2]
```diff
@@ -2,7 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette.by import By
+try:
+    from marionette.by import By
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_dsds_cell_data.py]: Import nou used and cam be removed

File: tests/python/gaia-ui-tests/gaiatest/apps/calendar/app.py [hunk 3]
```diff
@@ -2,12 +2,21 @@
...
-from marionette.by import By
-from marionette.errors import NoSuchElementException
-from marionette.errors import StaleElementException
-from marionette.marionette import Actions
-from marionette.wait import Wait
+try:
+    from marionette import expected
+    from marionette.by import By
+    from marionette.errors import (NoSuchElementException,
+                                   StaleElementException)
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/calendar/app.py]: This looks nice :heart:

**Comment by try-server-hook** [comment]
[AutomatedTester David Burns (AutomatedTester)](https://github.com/AutomatedTester) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=57725e68b29c)

**Pull request merged**